### DISCUSSION TITLE
How to add HTTP/PXE boot options in EFIShell?

### DISCUSSION THREAD

**Discussion by Rimits** [tianocore/edk2#4730] [Q&A]
The firmware in virtualbox, for example, don't included HTTP Boot, but HttpDxe.efi and HttpBootDxe.efi can be manually loaded in EFIShell. But then how can I add http boot options in EFIShell?
What is the correct syntax or way to add HTTP/PXE boot options in EFIShell?
Or is there a way to boot HTTP/PXE directly in EFIShell without adding a boot option?
![ksnip_20230810-000315](https://github.com/tianocore/edk2/assets/42634955/6c8e4432-1c7b-4737-951a-f2442054d9c2)

**Comment by bokkabonga** [comment]
Just stumpled upon this. Did you ever manage to resolve the problem?

I use the firmware in virt-manager/libvirtd and for me it doesn´t even show the NIC in the boot manager
